draft-06/draft-06.txt

```diff
@@ -584,6 +584,14 @@ Internet-Draft     Application-Level Profile Semantics      January 2021
    be sure to any exsiting 'format' does not conflict with the supplied
```

Comment:
perhaps this should read "they should be sure that any existing..."

Reply:
yep. thaniks, @malevy